src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -212,6 +215,8 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
         this.showOwnerDialog(this.rightClickedFile) }},
       { text: "Rename...", action:() => {
         this.showRenameField(this.rightClickedFile) }},
+      { text: "Tag Directory Contents", action:() => { 
```

Comment:
Personal preference: "Tag Directory Contents" should be "Tag Directory..." to show that it will spawn a modal/additional options